azkaban-common/src/main/java/azkaban/executor/Status.java

```diff
@@ -57,6 +58,9 @@ public enum Status {
   public static final ImmutableSet<Status> RESTARTABLE_TERMINAL_STATUSES =
       ImmutableSet.of(Status.EXECUTION_STOPPED, Status.FAILED);
 
+  public static final ImmutableSet<Status> TERMINATING_STATUSES = ImmutableSet.of(Status.SUCCEEDED,
```

Comment:
Based on other names in the class, should this not be TERMINAL_STATUSES?

Reply:
Makes sense. Let me rename

Reply:
DONE